frontend/src/worksheets/UpdateWorkSheetForm.js

```diff
@@ -24,18 +25,20 @@ function UpdateWorkSheetForm({
   title,
   worksheet,
 }) {
+  const [date, setDate] = useState(new Date())
```

Comment:
Jobb lenne ezt is Formikkal kezelni, ha szükséges írjunk egy Formik-os wrappert a CalendarDropDown köré, ami a useField-et használja. Elképzelhető, hogy az onSubmit hamarabb fut le, mint a state frissítése...